types/text-buffer/tslint.json

```diff
@@ -1,11 +1,12 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        // Custom rules.
         "class-name": true,
-        "indent": [true, "tabs"],
+        "indent": [true, "spaces", 4],
```

Comment:
I wouldn't change existing formatting settings without a reason.

Reply:
I originally chose tabs for these definitions whenever I thought editorconfig files were fine within the repository. I'd prefer spaces now that those files are gone, but if I really have to convert back to tabs then I can.

There aren't exactly any other authors to care about the switch over.

Reply:
OK, I was just worried that the next PR would change it back. If you're the only author then it's fine.